README.md

```diff
@@ -78,3 +82,28 @@ The endpoint takes no inputs and returns data in the following format:
   }
 ]
 ```
+
+### Adding Extra Routes
+
+You can do this with the ```ln_endpoints_data_routes``` filter as follows:
+
+```php
+add_filter( 'ln_endpoints_data_routes', [ __CLASS__, 'add_extra_routes' ] );
```

Comment:
I think this example requires a class can we specify that or just use a plain function instead.